Packs/HealthCheck/Scripts/HealthCheckLicenseData/HealthCheckLicenseData.py

```diff
@@ -16,65 +16,63 @@
+            usedUsers = python_dict['usedUsers']
+        elif 'soar' in python_dict['license']:
+            usedUsers = python_dict['license']['soar']['usedUsers']
+        else:
+            usedUsers = python_dict['license']['usedUsers']
 
-            if 'usedUsers' in python_dict:
-                usedUsers = python_dict['usedUsers']
-            elif 'soar' in python_dict['license']:
-                usedUsers = python_dict['license']['soar']['usedUsers']
-            else:
-                usedUsers = python_dict['license']['usedUsers']
+        if 'type' in python_dict:
+            licenseType = python_dict['type']
+        elif 'soar' in python_dict['license']:
+            licenseType = python_dict['license']['soar']['type']
+        elif 'types' in python_dict['license']:
+            licenseType = python_dict['license']['types']['soar']
+            TimLicenseType = python_dict['license']['types']['tim']
+        else:
+            licenseType = python_dict['license']['type']
 
-            if 'type' in python_dict:
-                licenseType = python_dict['type']
-            elif 'soar' in python_dict['license']:
-                licenseType = python_dict['license']['soar']['type']
-            elif 'types' in python_dict['license']:
-                licenseType = python_dict['license']['types']['soar']
-                TimLicenseType = python_dict['license']['types']['tim']
-            else:
-                licenseType = python_dict['license']['type']
+        if 'id' in python_dict:
+            uid = python_dict['id']
+        elif 'soar' in python_dict['license']:
+            uid = python_dict['license']['soar']['id']
+        else:
+            uid = python_dict['license']['id']
 
-            if 'id' in python_dict:
-                uid = python_dict['id']
-            elif 'soar' in python_dict['license']:
-                uid = python_dict['license']['soar']['id']
-            else:
-                uid = python_dict['license']['id']
+        demisto.executeCommand("setIncident", {
+            'healthcheckpermittedusers': permittedUsers,
+            'healthcheckusedusers': usedUsers,
+            'xsoarcustomername': customer,
+            'xsoarlicense': licenseType,
+            'xsoarlicensevalidtill': validTil,
+            'xsoartelemetryuuid': uid,
+        })
 
-            demisto.executeCommand("setIncident", {
-                'healthcheckpermittedusers': permittedUsers,
-                'healthcheckusedusers': usedUsers,
-                'xsoarcustomername': customer,
-                'xsoarlicense': licenseType,
-                'xsoarlicensevalidtill': validTil,
-                'xsoartelemetryuuid': uid,
-            })
-
-            if TimLicenseType:
-                demisto.executeCommand("setIncident", {'xsoartimlicensetype': TimLicenseType})
+        if TimLicenseType:
+            demisto.executeCommand("setIncident", {'xsoartimlicensetype': TimLicenseType})
```

Comment:
please add it to the command execution above (there is no reason to run it twice).